build/localmode/RunSamples.cmd

```diff
@@ -13,6 +19,13 @@ if "%1" == "" (
         set VERBOSE="verbose"
         @echo [RunSamples.cmd] VERBOSE is !VERBOSE!
     )
+
```

Comment:
RunSamples already do too many things. I am not sure if we want to add more to it. For running examples instructions could just involve 3 steps - 
* the first step is optional to run precheck.cmd to download & set env. variable for spark, hadoop etc. if users do not have them installed
* the second step is for folks who skip step #1 - to set env. variable for spark, hadoop
* third step is to run a "sparkclr-submit ...." command

After for M1 or after, we can make these steps easy to follow by having a script called RunExample.cmd